documentation/frontend/code-style.md

```diff
@@ -60,3 +62,9 @@ Classes for different flavors of API errors are established in errors.ts, use th
 ### Snapshots & Page Component Tests
 
 We have observed that running typical Jest unit tests on dynamic Next JS page level components is complicated and generally does not provide much value. When creating any new pages, feel free to rely on snapshot based tests rather than functional tests for these components.
```

Comment:
```suggestion
We have observed that running typical Jest unit tests on dynamic Next.js page level components is complicated and generally does not provide much value. When creating any new pages, feel free to rely on snapshot based tests rather than functional tests for these components.
```

Out of scope extreme nit, Next.js spelling / case